react-frontend/src/components/CaseStudies/horizontalinfocard.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import 'antd/dist/antd.css';
+import { Card } from 'antd';
+import '../../css/card.css';
+
+function HorizontalInfoCard(props) {
```

Comment:
Jon might have said the same thing but I think were using const declarations. const HorizontalInfoCard = () => {